shared/chat/conversation/normal/container.tsx

```diff
@@ -1,34 +1,36 @@
 import * as React from 'react'
-import * as Types from '../../../constants/types/chat2'
 import * as Constants from '../../../constants/chat2'
-import * as WaitingConstants from '../../../constants/waiting'
 import * as Chat2Gen from '../../../actions/chat2-gen'
 import * as Tracker2Gen from '../../../actions/tracker2-gen'
 import * as RouteTreeGen from '../../../actions/route-tree-gen'
-import Normal from '.'
 import * as Container from '../../../util/container'
+import Normal from '.'
+import type * as Types from '../../../constants/types/chat2'
 import {indefiniteArticle} from '../../../util/string'
 
 type Props = {conversationIDKey: Types.ConversationIDKey}
 
 const NormalWrapper = React.memo((props: Props) => {
   const {conversationIDKey} = props
   const [focusInputCounter, setFocusInputCounter] = React.useState(0)
-  const [scrollListDownCounter, setScrollListDownCounter] = React.useState(0)
```

Comment:
a downside of using these as numbers is that it forces a render when at the other end its just doing some dom manipulation, this way we can skip a render when this fires